bindings/pyroot/src/Utility.cxx

```diff
@@ -254,6 +254,26 @@ Bool_t PyROOT::Utility::AddToClass( PyObject* pyclass, const char* label, PyCall
    return kTRUE;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Add the given function with name 'func' to the class as read-only property 'property'.
+
+Bool_t PyROOT::Utility::AddProperty(PyObject *pyclass, const char *func, const char *property)
+{
+   PyObject *builtin = PyImport_ImportModule("__builtin__");
```

Comment:
I was doing a bit of research about how to set Python properties from C using the Python-C API, and I found [this](https://docs.python.org/2/c-api/structures.html#c.PyGetSetDef).

Reply:
I've seen the doc you referenced in the link as well, but this applies for adding new types in C extensions. I do not think that we have access to the data structure needed to use this. Have a look [here](https://docs.python.org/3/extending/newtypes.html).

They use the struct `PyGetSetDef` in the struct `PyTypeObject` to define the getters and setters. I don't think that this works for us or do you see where we could manipulate this information?